ai/cryptocom-ai-agent-service/src/services/agent/agent.service.ts

```diff
@@ -31,7 +31,7 @@ import {
 Client.init({
   chain: CronosZkEvm.Testnet,
   apiKey: process.env.EXPLORER_API_KEY!,
-  provider: 'http://localhost:5173',
+  provider: 'http://localhost:5173', //magic link
```

Comment:
we can remove the // comment

Reply:
maybe we should consider using process.env because http will be picked up as a security issue by sec scan

Reply:
ok